models/point_of_interest.py

```diff
@@ -51,9 +51,9 @@ def ensure_organizations_as_list(cls, organizations: Any) -> List[str]:
     def sanitize_url(cls, url: str) -> str:
         return sanitise_url(url)
 
-    @validator('phone', pre=True)
-    def sanitize_phone(cls, phone: str) -> str:
-        return sanitise_phone(phone)
+    @validator('phone')
```

Comment:
Did you test this with spreadsheet conversion?

Reply:
No. 
But we've got an existing Unit Test for it. As you can see, it makes the CircleCI broken.

By the way, I'm refactoring these parts as well.

Reply:
Ok, please, test manually too